yt/frontends/ramses/fields.py

```diff
@@ -33,7 +33,7 @@
 rho_units = "code_density"
 vel_units = "code_velocity"
 pressure_units = "code_pressure"
-ener_units = "code_mass * code_velocity**2 / code_time**2"
+ener_units = "code_mass * code_velocity**2"
```

Comment:
good catch! it looks like this was only used for the `BH_esave` field, which I guess is for black hole particles, so I guess no one noticed until not because of that. Any chance this breaks the units for that field though or were those wrong to begin with?

Reply:
They were wrong to begin with ; since it has only been implemented for a couple of months, I guess noone ran into the issue.